vendor/vendor.json

```diff
@@ -0,0 +1,75 @@
+{
+	"comment": "",
+	"ignore": "test",
+	"package": [
+		{
+			"checksumSHA1": "FUhczVSvt/VQAlWfzATo1YLez2E=",
+			"path": "github.com/Financial-Times/go-fthealth/v1a",
+			"revision": "f9074331b3ab48bca0e6d0afc211a673d88ac925",
+			"revisionTime": "2016-06-16T09:04:59Z"
+		},
+		{
+			"checksumSHA1": "yceqWuIbFzntmSHl66OwNoP/c6E=",
+			"path": "github.com/Financial-Times/message-queue-go-producer/producer",
```

Comment:
yep, thank you for that, I will add specific versions on libs where possible :+1: